server/controller/Entries.js

```diff
@@ -3,6 +3,7 @@ import EntryModel from '../model/EntriesModel';
 const Entries = {
     create(req, res) {
```

Comment:
req, res?
Please use better descriptions for your variables `request`, `response`

Reply:
ok, thanks am working on it